apps/sriov-kernel-nse/nse.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nse
+  labels:
+    app: nse
+spec:
+  selector:
+    matchLabels:
+      app: nse
+  template:
+    metadata:
+      labels:
+        app: nse
+    spec:
+      containers:
+        - name: nse
+          image: rrandom312/cmd-nse-icmp-responder:testing
```

Comment:
Should we use here nsm hub?

Reply:
Removed SR-IOV kernel examples, because ICMP responder is still not ready.